README.md

```diff
@@ -302,6 +304,51 @@ options:
                         Set the logging level for stderr logging
 ```
 
+### AI-assisted academic authoring
+
+The `manubot ai-revision` command ... COMPLETE
+
+<!-- test codeblock contains output of `manubot ai-revision --help` -->
```

Comment:
Based on test failures, looks like the help output here could be stale?

Reply:
I just [fixed it](https://github.com/manubot/manubot/pull/360/commits/b41cef8b298556946212d12ef481af0176d0a9ea). However, I think the expected, split output adds a new line in the middle of a parameter name (`--model-type`) which doesn't look great. But it's a very minor detail.

Reply:
The complete instructions to use the tool are in manubot/rootstock#484. A basic setup (such as setting `OPENAI_API_KEY` is missing here). Maybe we should add a link to rootstock's `USAGE.md`?

Reply:
We could play with https://docs.python.org/3/library/argparse.html#formatter-class but probably not worth the hassle.